### ISSUE TITLE
num_axes() <= 4 (5 vs. 4) Cannot use legacy accessors on Blobs with > 4 axes

### ISSUE THREAD

**Issue by EthanjWang** [facebook/C3D#248]
F0419 10:50:16.235648 23639 blob.hpp:140] Check failed: num_axes() <= 4 (5 vs. 4) Cannot use legacy accessors on Blobs with > 4 axes.
*** Check failure stack trace: ***
    @     0x7f690a5ca5cd  google::LogMessage::Fail()
    @     0x7f690a5cc433  google::LogMessage::SendToLog()
    @     0x7f690a5ca15b  google::LogMessage::Flush()
    @     0x7f690a5cce1e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f690acc3096  caffe::CuDNNReLULayer<>::Reshape()
    @     0x7f690ada3d29  caffe::Net<>::Init()
    @     0x7f690ada648e  caffe::Net<>::Net()
    @     0x7f690adbcb1c  caffe::Solver<>::InitTrainNet()
    @     0x7f690adbdf25  caffe::Solver<>::Init()
    @     0x7f690adbe24f  caffe::Solver<>::Solver()
    @     0x7f690ad7af31  caffe::Creator_SGDSolver<>()
    @           0x40ade8  train()
    @           0x407548  main
    @     0x7f6908c49ac0  __libc_start_main
    @           0x407e19  _start
    @              (nil)  (unknown)
./finetuning_ucf101.sh: line 7: 23639 Aborted                 (core dumped) GLOG_log_dir="./LOG_TRAIN" ../../build/tools/caffe.bin train --solver=solver_r2.prototxt --weights=c3d_resnet18_sports1m_r2_iter_2800000.caffemodel --gpu=0

I got this error. Can anyone tell me how to solve it ?
Thanks for your time and reply.

**Comment by dutran** [comment]
Turn off CUDNN option and re-compile. See also https://github.com/facebook/C3D/issues/244

**Comment by xasd2003** [comment]
@dutran  How to turn off CUDNN option and re-compile？

**Comment by Jessespace** [comment]
I solve it by modifying the limitation for num_axes(),I train it on whole UCF-101,But,i got the loss about 87.3365,why ?
